packages/melange-moment/melange-moment.0.4.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Melange bindings for Moment.js"
+maintainer: "Ahrefs"
+authors: "Jiayu Liu <jimexist@gmail.com>"
+license: "MIT"
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/melange-moment"
+bug-reports: "https://github.com/ahrefs/melange-moment"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml"
+  "melange" {>= "3.0.0"}
+  "reason"
+  "melange-jest" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```
Due to 
```
#=== ERROR while compiling melange-moment.0.4.0 ===============================#
# context              2.2.0~beta3~dev | linux/s390x | ocaml-base-compiler.5.2.0 | pinned(https://github.com/ahrefs/melange-moment/releases/download/0.4.0/melange-moment-0.4.0.tbz)
# path                 ~/.opam/5.2/.opam-switch/build/melange-moment.0.4.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p melange-moment -j 3 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/melange-moment-7-fc0559.env
# output-file          ~/.opam/log/melange-moment-7-fc0559.out
### output ###
# File "dune", line 14, characters 8-11:
# 14 |    (run npx jest --config ../../%{project_root}/jest.config.cjs)
#              ^^^
# Error: Program npx not found in the tree or in PATH
#  (context: default)
```

Otherwise we need a new conf-npx to depend on for the tests

Reply:
I don't understand these `EAI_AGAIN` errors... Skipping the tests for now, as we run them already in our repo CI and we also skipped them in previous releases: https://github.com/ocaml/opam-repository/pull/24653.